open-reac/src/test/java/com/powsybl/openreac/OpenReacParametersTest.java

```diff
@@ -40,7 +43,55 @@ public void testObjectiveIntegrityChecks() {
     }
 
     @Test
-    public void testParametersIntegrityChecks() {
```

Comment:
Can you add an export readable by AMPL files to show the default values and a change in the default values? But I don't know how you can check the results without reader :-)